tutorials/snake/steps/step-1.md

```diff
@@ -2,22 +2,22 @@
 
 ## Basic file structure
 
-In this tutorial we will program using a Clojure-like syntax and an EDN-like format (JSON equivalent for Clojure). The similarity of Chainblock's syntax to Clojure means more expressive code in fewer lines (as compared to traditional programming languages like Javascript or Python). Additionally, the code is more focused on data which makes it easier for beginners to grasp.
+In this tutorial, we will program using a Clojure-like syntax and an EDN-like format (JSON equivalent for Clojure). The similarity of Shards' syntax to Clojure means more expressive code in fewer lines (as compared to traditional programming languages like Javascript or Python). Additionally, the code is more focused on data which makes it easier for beginners to grasp.
 
-A simple program in Chainblocks would look like this:
+A simple program in Shards would look like this:
 
 === "EDN"
 
     ```clojure linenums="1"
-    ; define a chain named 'main-chain'
-    (defchain main-chain
+    ; define a wire named 'main-wire'
+    (defchain main-wire
```

Comment:
```suggestion
    (defwire main-wire
```

Reply:
Fixed with [this](https://github.com/fragcolor-xyz/shards-examples/pull/11/commits/f0f4adf90754e0979be9b4b36ae91214172afdb4) and [this](https://github.com/fragcolor-xyz/shards-examples/pull/11/commits/94768e3dda29551b8f04afb8e92ac2c36af5534c) commit